ps_contactinfo.php

```diff
@@ -141,6 +141,7 @@ public function getContent()
             'type' => 'switch',
             'label' => $this->trans('Display email address', array(), 'Admin.Actions'),
             'name' => 'PS_CONTACT_INFO_DISPLAY_EMAIL',
+            'desc' => $this->trans('Your theme needs to be compatible with this feature', array(), 'Modules.Contactinfo.Admin'),
             'values' => array(
```

Comment:
Hi @LouiseBonnard , I was not so sure of which domain translation to use here ?

Reply:
Will this PR add a configuration page to the 'Contact Information' module? I don't think there is any currently. If yes, you got the right domain. ;-)

Reply:
yes, we added a configuration page, and this translation will be there, it just hasn't been released yet.

Good then ! thanx